ui_framework/components/button/submit_button.test.js

```diff
@@ -24,6 +24,7 @@ describe('KuiSubmitButton', () => {
           aria-label="aria label"
           className="testClass1 testClass2"
           data-test-subj="test subject string"
+          type="submit"
```

Comment:
I'm sorry, I forgot that this component will always apply the `type="submit"` attribute. So we can remove this from this test.